projects/kit/components/tabs/tabs.directive.ts

```diff
@@ -51,14 +54,12 @@ export class TuiTabsDirective implements AfterViewChecked {
     }
 
     public ngAfterViewChecked(): void {
```

Comment:
can we move this to another lifecycle hook? for example `ngOnInit`, will this work?

Reply:
I think not, because we have to update the state after every rendering, and not just after initialization. But if anyone has any ideas on how to implement it without `afterNextRender`, I’d be glad for a hint